src/app/child-dev-project/children/aser/aser-component/aser.component.ts

```diff
@@ -31,18 +27,4 @@ export class AserComponent extends RelatedEntitiesComponent<Aser> {
     { id: "bengali", visibleFrom: "md" },
     { id: "remarks", visibleFrom: "md" },
   ];
-
-  constructor(
-    private childrenService: ChildrenService,
-    entityMapper: EntityMapperService,
-    entityRegistry: EntityRegistry,
-    screenWidthObserver: ScreenWidthObserver,
-    filterService: FilterService,
-  ) {
-    super(entityMapper, entityRegistry, screenWidthObserver, filterService);
-  }
-
-  override getData() {
-    return this.childrenService.getAserResultsOfChild(this.entity.getId());
```

Comment:
I am confused: Did `getAserResultsOfChild()` not exist anymore before this PR also? Can't find it in master ...

If so, let's remove this component completely and migrate for existing servers to just use "RelatedEntities" directly

Reply:
No I recently removed it because it only used the entity mapper and was only used at one place. I did not remove the component because we still have the predefined config, but this can probably also be migrated.